troposphere/codebuild.py

```diff
@@ -161,6 +161,35 @@ class VpcConfig(AWSProperty):
     }
 
 
+class WebhookFilterRule(AWSProperty):
+    props = {
+        'JsonPath': (str, True),
```

Comment:
All string properties should be defined with the type `basestring` instead of `str`.